### DISCUSSION TITLE
Tool versions?

### DISCUSSION THREAD

**Discussion by karboski** [NVIDIA/MatX#484] [Q&A] [answered]
Any known-good combination of distro, gcc and cuda? I'm using matx successfully but can't build the tests.

**Comment by cliffburdick** [comment, marked as answer]
Hi @karboski , can you let us know the errors and what versions you're using? We typically test on CUDA 11.8 and CUDA 12.2.1 with Ubuntu 20.04 and 22.04 with the native host compiler. If there's another combination that's not working we can test it out.

**Reply by karboski** [reply]
My project uses clang, and I was hoping to send in patches to that effect, but wanted to run your tests on gcc first. I can make a VM with the versions you suggested.

In any case:

Centos 8 stream, NVIDIA 12.2.140, GNU 11.2.1

/tmp/MatX/include/matx/transforms/filter.h:56:22: error: expected unqualified-id before ‘[’ token
   matxFilter_t([[maybe_unused]] OutType &o, const InType &i,

**Comment by cliffburdick** [comment]
That looks like c++17 support is not enabled. We can try to reproduce it on our side and get back to you. Clang was not working last we tested and there are some issues we're aware of to get it working. We can try again and see where the latest clang is at.

**Comment by cliffburdick** [comment]
Hi @karboski, are you using a centos container image? I'm assuming not since I can't find that image in our repo.

**Reply by karboski** [reply]
It's an old package, I wouldn't worry about it. Ubuntu is working fine.
